<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1486</width>
    <height>1063</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: #2E073F;
border-radius: 10px</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_top">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>80</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>src-hmi/logo.png</normaloff>src-hmi/logo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_9">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>82</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>82</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:#7A1CAC;

border-radius:30px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">font-size:48px;
color:#FFFFFF;
</string>
               </property>
               <property name="text">
                <string>Swarm Robotics Interface</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_8">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="cb_list_ports">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:#EBD3F8;
color: #000000
</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_connect">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">

QPushButton{
background-color:#7A1CAC;
border-radius:10px;
color:#FFFFFF;
font-size:20px
}

QPushButton:hover{

background-color:#3B0E53;
border-radius:10px;

}</string>
               </property>
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_update">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>10</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">

QPushButton{
background-color:#7A1CAC;
border-radius:10px;
color:#FFFFFF
}

QPushButton:hover{

background-color:#3B0E53;
border-radius:10px;

}</string>
               </property>
               <property name="text">
                <string>Update</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>1451</width>
            <height>921</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>100</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="minimumSize">
                <size>
                 <width>674</width>
                 <height>448</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>674</width>
                 <height>448</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:#7A1CAC;

border-radius:42px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="temp_layout"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="minimumSize">
                <size>
                 <width>673</width>
                 <height>448</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>673</width>
                 <height>448</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:#7A1CAC;

border-radius:42px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QVBoxLayout" name="hum_layout"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>100</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_5">
               <property name="minimumSize">
                <size>
                 <width>961</width>
                 <height>447</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>961</width>
                 <height>447</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:#7A1CAC;

border-radius:42px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <layout class="QVBoxLayout" name="map_layout"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>367</width>
                 <height>311</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>367</width>
                 <height>311</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:#7A1CAC;

border-radius: 42px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QLabel" name="label_2">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>0</y>
                  <width>261</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:32px;
color:#FFFFFF</string>
                </property>
                <property name="text">
                 <string>Rover Location:</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_3">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>60</y>
                  <width>111</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:24px;
color: #FFFFFF;</string>
                </property>
                <property name="text">
                 <string>Rover 1:</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_4">
                <property name="geometry">
                 <rect>
                  <x>100</x>
                  <y>90</y>
                  <width>91</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:20px;
color: #FFFFFF;</string>
                </property>
                <property name="text">
                 <string>Latitude:</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_5">
                <property name="geometry">
                 <rect>
                  <x>100</x>
                  <y>120</y>
                  <width>101</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:20px;
color: #FFFFFF;
</string>
                </property>
                <property name="text">
                 <string>Longitude:</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_6">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>150</y>
                  <width>131</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:24px;
color: #FFFFFF;</string>
                </property>
                <property name="text">
                 <string>Rover 2:</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_7">
                <property name="geometry">
                 <rect>
                  <x>110</x>
                  <y>190</y>
                  <width>91</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:20px;
color: #FFFFFF;</string>
                </property>
                <property name="text">
                 <string>Latitude:</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_8">
                <property name="geometry">
                 <rect>
                  <x>110</x>
                  <y>220</y>
                  <width>101</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:20px;
color: #FFFFFF;
</string>
                </property>
                <property name="text">
                 <string>Longitude:</string>
                </property>
               </widget>
               <widget class="QLabel" name="lat_val_1">
                <property name="geometry">
                 <rect>
                  <x>200</x>
                  <y>90</y>
                  <width>141</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:20px;
color: #FFFFFF;</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLabel" name="lon_val_1">
                <property name="geometry">
                 <rect>
                  <x>200</x>
                  <y>120</y>
                  <width>141</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:20px;
color: #FFFFFF;
</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLabel" name="lon_val_2">
                <property name="geometry">
                 <rect>
                  <x>210</x>
                  <y>220</y>
                  <width>131</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:20px;
color: #FFFFFF;
</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLabel" name="lat_val_2">
                <property name="geometry">
                 <rect>
                  <x>210</x>
                  <y>190</y>
                  <width>131</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-size:20px;
color: #FFFFFF;</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
